docs/tempo/website/operations/generic_forwarding.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Generic forwarding
+weight: 13
+---
+
+# Generic forwarding
+
+Generic forwarding is a feature that allows asynchronous replication of ingested traces. The distributor will write received spans to both the ingester and defined endpoints if enabled. This feature works in a "best-effort" manner, meaning that no retries happen if an error occurs during replication. 
+
+Warning: Generic forwarding does not work retroactively. The distributor will replicate only freshly ingested spans once this feature is enabled.
+
+## Configuration
+
+Enabling generic forwarding requires the configuration of two sections. First, define a list of forwarders in the `distributor` section. Each forwarder must specify a unique `name`, supported `backend`, and backend-specific configuration. Second, reference these forwarders in the `overrides` section. This allows for fine-grained control over forwarding and makes it possible to enable this feature globally or on a per-tenant basis.
```

Comment:
Should this paragraph really be a numbered procedure?

Reply:
Hmm, you can't have (2) if you didn't set up (1) in the first place.